QUICKSTART.md

```diff
@@ -0,0 +1,279 @@
+  - [c. Onboarding the clusters](#b-onboarding-the-clusters)
+  - [d. Create and deploy the Apache Server workload into florin and guilder clusters](#c-Create-and-deploy-the-nginx-workload-into-florin-and-guilder-clusters)
+- [3. Cleanup the environment](#3-Cleanup-the-environment)
+
+
+This guide is intended to show how to quickly bring up a **KubeStellar** environment with its dependencies from a binary release.
+
+## 1. Install and run **KubeStellar**
+
+KubeStellar works in the context of kcp, so to use KubeStellar you also need kcp. Download the kcp and **KubeStellar** binaries and scripts into a `kubestellar` subfolder in your current working directory using the following command:
+
+```shell
+bash <(curl -s https://raw.githubusercontent.com/francostellari/edge-mc/main/hack/kubestellar-bootstrap.sh) --kcp-version v0.11.0 --kubestellar-version v0.1.0 --folder . --create-folder
+export PATH="$PATH:$(pwd)/kcp/bin:$(pwd)/kubestellar/bin"
+export KUBECONFIG="$(pwd)/.kcp/admin.kubeconfig"
+```
+
+Check that `KubeStellar` is running:
+
+First, check that controllers are running with the following command:
+
+```shell
+ps aux | grep -e mailbox-controller -e placement-translator -e scheduler
+```
+
+which should yield something like:
+
+```console
+user     1892  0.0  0.3 747644 29628 pts/1    Sl   10:51   0:00 mailbox-controller --inventory-context=root --mbws-context=base -v=2
+user     1902  0.3  0.3 743652 27504 pts/1    Sl   10:51   0:02 scheduler -v 2 --root-user kcp-admin --root-cluster root --sysadm-context system:admin --sysadm-user shard-admin
+user     1912  0.3  0.5 760428 41660 pts/1    Sl   10:51   0:02 placement-translator --allclusters-context system:admin -v=2
+```
+
+Second, check that the Edge Service Provider Workspace (`espw`) is created with the following command:
+
+```shell
+kubectl ws tree
+```
+
+which should yield:
+
+```console
+kubectl ws tree
+.
+└── root
+    ├── compute
+    └── espw
+```
+
+## 2. Example deployment of Apache HTTP Server workload into two kind local clusters
+
+### a. Stand up a local florin and guilder kind clusters
+
+Create the first edge cluster:
+
+```shell
+kind create cluster --name florin --config examples/florin-config.yaml
+```  
+
+Create the second edge cluster:
+
+```shell
+kind create cluster --name guilder --config examples/guilder-config.yaml
+```  
+
+### b. Create a KubeStellar Inventory Management Workspace (IMW) and Workload Management Workspace (WMW)
+
+IMW are used by KubeStellar to store inventory objects (sync targets and placement). Create an IMW named `example-imw` with the following command:
+
+```shell
+kubectl config use-context root
+kubectl ws root
+kubectl ws create "example-imw"
+```
+
+WMW are used by KubeStellar to store workloads and edge placement objects. Create an WMW named `example-wmw` in a `my-org` workspace with the following command:
```

Comment:
(1) I would qualify "workloads" here, since the WMW holds only some aspects of the workloads.  All the aspects that are specific to _running_ the workload are not in the WMW.  So here I would say something like "workload prescriptions" or "workload descriptions".

(2) Since the purpose of a QuickStart is to get the user ready to use the platform, I would point out here that the platform does NOT require that the WMW be created before the edge clusters are on-boarded.   Perhaps with langauge like the following.

> A WMW does not have to be created before the edge cluster is on-boarded; the WMW only needs to be created before content is put in it.

Reply:
Fixed